server/src/main/java/org/opensearch/rest/action/cat/RestNodesAction.java

```diff
@@ -525,4 +528,21 @@ Table buildTable(
     private short calculatePercentage(long used, long max) {
         return max <= 0 ? 0 : (short) ((100d * used) / max);
     }
+
+    /**
+     * Parse the deprecated request parameter 'master_timeout', and add deprecated log if the parameter is used.
+     * It also validates whether the value of 'master_timeout' is the same with 'cluster_manager_timeout'.
+     * Remove the method along with MASTER_ROLE.
+     * @deprecated As of 2.0, because promoting inclusive language.
+     */
+    @Deprecated
+    private void parseDeprecatedMasterTimeoutParameter(ClusterStateRequest clusterStateRequest, RestRequest request) {
+        final String deprecatedTimeoutParam = "master_timeout";
+        final String clusterManagerTimeoutParam = "cluster_manager_timeout";
+        clusterStateRequest.masterNodeTimeout(request.paramAsTime(deprecatedTimeoutParam, clusterStateRequest.masterNodeTimeout()));
+        if (request.hasParam(deprecatedTimeoutParam)) {
+            deprecationLogger.deprecate("cat_nodes_master_timeout_parameter", MASTER_TIMEOUT_DEPRECATED_MESSAGE);
+            request.validateParamValuesAreEqual(deprecatedTimeoutParam, clusterManagerTimeoutParam);
+        }
```

Comment:
Two nitpicks here:

1. Setting the masterNodeTimeout should occur within the if-condition
2. The validation could occur before setting the value

Reply:
Great suggestion! Now the procedure is more reasonable 😄. I modified the code in the commit https://github.com/opensearch-project/OpenSearch/pull/2435/commits/399f2d90568a47f42d8c65b1e306a867666a30d9